app.js

```diff
@@ -90,16 +90,18 @@ app.get('/api/wish_list/get_users', wishListController.get_users);
 // Possible trades
 app.post('/api/possible_trades/add', possibleTradesController.add_relation);
 app.post('/api/possible_trades/remove', possibleTradesController.remove_relation);
-app.post('/api/possible_trades/remove_have', possibleTradesController.remove_relation_have);
-app.post('/api/possible_trades/remove_want', possibleTradesController.remove_relation_want);
+// app.post('/api/possible_trades/remove_have', possibleTradesController.remove_relation_have);
+// app.post('/api/possible_trades/remove_want', possibleTradesController.remove_relation_want);
 app.get('/api/possible_trades/get_book_wants', possibleTradesController.get_book_wants);
 
 /*
  * Tests
  */
-const ownedBooksTest = require('./tests/models/owned_books').test();
-const possibleTradesTest = require('./tests/models/possible_trades').test();
-const usersTest = require('./tests/models/users').test();
+const initTest = require('./models/init').create_tables(function(){
+    const ownedBooksTest = require('./tests/models/owned_books').test();
+    const possibleTradesTest = require('./tests/models/possible_trades').test();
+    const usersTest = require('./tests/models/users').test();
+});
 
```

Comment:
moved all testing to another file. Now just need to call one function in app.js to test everything